c8/tenant.py

```diff
@@ -358,6 +359,26 @@ def response_handler(resp):
     # User Management #
     ###################
 
+    def set_user_api_url(self, request, response_handler):
+        """
+        Sets the user management api endpoints to /_api/ instead of 
+        /_fabric/{fabric}/_api/ since in user management is done at a tenant
+        level not fabric level.
+        """
+        # We need to rewrite the connection's standard URL prefix
+        # because the user management api call does not expect
+        # the '_fabric/{fabric}' part of the URL. So we rewrite it here
+        # temporarily to do the user management call, then set it back
+        oldconnprefix = self._conn.url_prefix
+        url = self._conn.url
+
+        self._conn.set_url_prefix(url + "/_api")
```

Comment:
Maybe refactor `_execute` so it takes an alternative prefix as a parameter. The problem with changing it like this is that if something else tries to use `_conn` while this is running, it'll cause that other call to use the wrong prefix. Also, if there's any exception during `_execute`, the code that switches it back won't run and the class will be left in a "broken" state.

Reply:
@ricardo-macrometa I have updated the same. may you review again?